### PR TITLE
Add cursor support

### PR DESCRIPTION
**Pull request by fsotomsk**
Added compatible `->cursor()` for query builder

### CODE REVIEW

File: src/Jenssegers/Mongodb/Query/Builder.php [hunk 1]
```diff
@@ -209,12 +215,25 @@ public function get($columns = [])
         return $this->getFresh($columns);
     }
 
+    /**
+     * @inheritdoc
+     */
+    public function cursor($columns = [])
```
Comment (Smolevich) [inline src/Jenssegers/Mongodb/Query/Builder.php:221]: Add test case for this function

File: tests/QueryBuilderTest.php [hunk 2]
```diff
@@ -759,4 +760,23 @@ public function testHintOptions()
...
+            ['name' => 'spork', 'tags' => ['sharp', 'pointy', 'round', 'bowl']],
+            ['name' => 'spoon', 'tags' => ['round', 'bowl']],
+        ];
+        DB::collection('items')->insert($data);
+
+        $results = DB::collection('items')->orderBy('_id', 'asc')->cursor();
+
+        $this->assertInstanceOf(LazyCollection::class, $results);
+        $i = 0;
+        foreach ($results as $result) {
```
Comment (Smolevich) [inline tests/QueryBuilderTest.php:777]:
```suggestion
        foreach ($results as $i => $result) {
```

Comment (fsotomsk) [reply tests/QueryBuilderTest.php]: Thanks)

**Comment by angelblanco** [comment]
Its this suitable to be merged in develop? Thanks!

**Pull request merged**